src/main/java/appeng/tile/crafting/TileCraftingStorageTile.java

```diff
@@ -36,24 +38,29 @@ protected ItemStack getItemFromTile( final Object obj )
 		final IBlocks blocks = AEApi.instance().definitions().blocks();
 		final int storage = ( (TileCraftingTile) obj ).getStorageBytes() / KILO_SCALAR;
 
+		Optional<ItemStack> is;
+
 		switch( storage )
 		{
 			case 4:
-				for( final ItemStack stack : blocks.craftingStorage4k().maybeStack( 1 ).asSet() )
+				is = blocks.craftingStorage4k().maybeStack( 1 );
+				if( is.isPresent() )
 				{
-					return stack;
+					return is.get();
```

Comment:
Oh, I haven't actually thought of adding a default branch and moving the orElseGet down there.
Will change this (I remember another location where I could use this, will find/fix it there too).